components/automate-ui/src/app/pages/+compliance/shared/reporting/suggestions.service.ts

```diff
@@ -15,10 +15,12 @@ export class SuggestionsService {
     private statsService: StatsService
   ) {}
 
+  selectedControlTagKey: string;
+
   getSuggestions(type: string, text: string, reportQuery: ReportQuery): Observable<any> {
     const url = `${CC_API_URL}/reporting/suggestions`;
     const formatted = this.statsService.formatFilters(reportQuery);
-    const body = {type, text, filters: formatted};
+    const body = {type, text, 'type_key': this.selectedControlTagKey, filters: formatted};
```

Comment:
```suggestion
        const body = {
          type,
          text,
          type_key: this.selectedControlTagKey,
          filters: formatted
        };
```

They've slipped through in some spots, but it was pointed out to me that the preferred route is not using the quotations